kolibri/content/api.py

```diff
@@ -24,6 +26,28 @@ class ChannelMetadataCacheViewSet(viewsets.ModelViewSet):
     def get_queryset(self):
         return models.ChannelMetadataCache.objects.all()
 
+    def destroy(self, request, pk=None):
```

Comment:
Easiest way is probably to add a permissions class to the ViewSet definition.

http://www.django-rest-framework.org/api-guide/permissions/#setting-the-permission-policy